regression/pages/studio/logout_studio.py

```diff
@@ -0,0 +1,19 @@
+"""
+Logout Page for Studio
+"""
+from bok_choy.page_object import PageObject
+from regression.pages.studio import BASE_URL
+
+
+class StudioLogout(PageObject):
+    """
+    Logged Out Page for Studio
+    """
+
+    url = BASE_URL
+
+    def is_browser_on_page(self):
+        """
+        Checks if we are on the correct page
+        """
+        return 'Welcome | edX Studio' in self.browser.title
```

Comment:
This could be a flaky condition. Is there a page object query you could use instead?